proto/cacheclient.proto

```diff
@@ -31,6 +32,14 @@ message _GetResponse {
   string message = 3;
 }
 
+message _DeleteRequest {
+  bytes cache_key = 1;
+}
+
+message _DeleteResponse {
+  ECacheResult result = 1;
```

Comment:
This field is unnecessary. Delete invocations either succeed (grpc OK) or fail/error (anything other than grpc OK)